src/components/BeerDetail.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+
+
+function BeerDetail(props) {
+
+    const { beerId } = useParams();
+    const [beersArr, setBeersArr] = useState([]);
```

Comment:
An empty array is always truthy and you aren't calling for an array and just One resource, so beersArr won't be a good name in here

```suggestion
    const [beer, setBeer] = useState(null);
```